Sources/docc/DocCDocumentation.docc/Reference Syntax/API Reference Syntax/Metadata.md

```diff
@@ -56,6 +56,17 @@ to add additional URLs for a page.
 > Note: Starting with version 6.0, @Redirected is supported as a child directive of @Metadata. In
 previous versions, @Redirected must be used as a top level directive.
 
+### Usage in documentation comments
+
+You can specify `@Metadata` configuration in documentation comments for the following customizations:
+
+- ``Available``
+- ``CustomMetadata``
+
+> Note: Don't specify an `@Metadata` directive in both the symbol's documentation comment and its documentation extension file.
+If you have one in each, DocC will pick the one in the documentation extension and discard the one in the documentation
+comment.
+
```

Comment:
It would be good to add an "Earlier Versions" aside, like on [the @Available documentation](https://www.swift.org/documentation/docc/available) to say that before Swift-DocC 6.2 (or 6.1 if we plan on cherry-picking this), @Metadata isn't supported in documentation comments.